templates/terraform.tfvars.openstack-template

```diff
@@ -4,7 +4,7 @@ cluster_prefix = "your-cluster-prefix" # Your cluster prefix
 kubeadm_token = "your-kubeadm-token" # Autogenerated kubeadm token (don't change it)
 floating_ip_pool = "your-pool-name"
 external_network_uuid = "external-net-uuid" # The uuid of the external network in the OpenStack tenancy
-boot_image = "kubenow-v050"
```

Comment:
same as above.

Reply:
Now master was built without problem on 050 so I guess the issue is gone - It might have been a temporary problem - The problem was that bootstrap script didn't find kubernetes.

Reply:
ok! Please put back 050 in the templates, and I'll merge!